### PR TITLE
add InfluxDB blog post link about Parquet catalog

### PR DESCRIPTION
**Pull request by charlesmahler**
adding a link to the blog posts resource section to a post about how InfluxDB is using Parquet for their new storage engine

### CODE REVIEW

**Comment by charlesmahler** [comment]
CC @shangxinli

**Comment by wgtmac** [comment]
Sorry for being late. Do you still want to merge this? @charlesmahler

**Comment by charlesmahler** [comment]
@wgtmac yes please, no worries about the delay

**Review by wgtmac — APPROVED** [review]
Thanks! LGTM. cc @gszadovszky @shangxinli

**Review by gszadovszky — APPROVED** [review]

File: content/en/docs/Learning Resources/Blog Posts/_index.md [hunk 1]
```diff
@@ -9,3 +9,5 @@ description: >
 * [Apache Parquet: Cost Efficiency @ Scale in Big Data File Format](https://eng.uber.com/cost-efficiency-big-data/)
 
 * [One Stone, Three Birds: Finer-Grained Encryption @ Apache Parquet](https://eng.uber.com/one-stone-three-birds-finer-grained-encryption-apache-parquet/)
+
+* [Designing a Parquet Catalog for InfluxDB IOx](https://www.influxdata.com/blog/designing-a-parquet-catalog-for-influxdb-iox/)
```
Comment (alamb) [inline content/en/docs/Learning Resources/Blog Posts/_index.md:13]: I am not sure the publishing process is supposed to work, but this blog post does not appear to be on the parquet.apache.org site https://parquet.apache.org/docs/learning-resources/blog-posts/

🤔 

Screenshot 2023-04-27 at 12 40 35 PM

Comment (wgtmac) [reply content/en/docs/Learning Resources/Blog Posts/_index.md:13]: Hmm... That's weird. I can see the link locally.

Comment (alamb) [reply content/en/docs/Learning Resources/Blog Posts/_index.md:13]: > Hmm... That's weird. I can see the link locally.

As in you can see it when the docs are built locally or you can see it when you open https://parquet.apache.org/docs/learning-resources/blog-posts/ locally 🤔

Comment (wgtmac) [reply content/en/docs/Learning Resources/Blog Posts/_index.md:13]: I can see it from the locally built site. I have no idea why it is missing from the published one.

**Pull request merged**